src/hooks/useNameValidation.ts

```
@@ -1,8 +1,6 @@
 import { useCallback, useEffect, useMemo, useRef, useState } from 'react'
-import { useQuery } from '@tanstack/react-query'
-import { convexQuery, useConvex } from '@convex-dev/react-query'
 import { debounce } from 'lodash-es'
-import { api } from 'convex/_generated/api'
+import { useSidebarItemMutations } from '~/hooks/useSidebarItemMutations'
```

ESLint

[error] `~/hooks/useSidebarItemMutations` import should occur after type import of `convex/sidebarItems/baseTypes`

(import/order)